sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ProcessBundleHandler.java

```diff
@@ -296,6 +296,7 @@ public void close() throws Exception {
     }
 
     @Override
+    @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
Suggest in line comment as to why - perhaps:

```// defer error handling on deregistering as that is the responsibility of the beamFnStateClient```

Reply:
Good catch. Done